.ci/templates/generate-result-links-markdown.sh

```diff
@@ -103,3 +109,8 @@ for experiment in $dsd_only_experiments; do
 
     echo "| $experiment (DSD only) | \\[[Profiling (DSD)]($dsd_continuous_profiler_url)\\] \\[[SMP Dashboard]($adp_smp_dashboard_url)\\] |"
 done
+
+checks_continuous_profiler_url=$(get_continuous_profiler_url "$checks_run_id" "$checks_start_time" "$checks_end_time" "quality_gates_idle_rss")
+checks_smp_dashboard_url=$(get_adp_smp_dashboard_url "$checks_run_id" "$dsd_run_id" "$checks_start_time" "$checks_end_time" "quality_gates_idle_rss")
+
+echo "| quality_gates_idle_rss | \\[[Profiling (Checks Agent)]($checks_continuous_profiler_url)\\] \\[[SMP Dashboard]($checks_smp_dashboard_url)\\] |"
```

Comment:
Maybe we change the table header (on line 86) to say `ADP Experiment Result Links` and then down here, we `echo` a similar table header that says `Checks Agent Experiment Result Links` since right now we'll have two rows with an identical `experiment` value of `quality_gates_idle_rss`, only differing in terms of the link text shown.

Reply:
https://github.com/DataDog/saluki/pull/618/commits/6a26a3b69e216ade872c018cc373be7b4b6ab2c2